### ISSUE TITLE
191230-01 - linux kernel v5.4.x images for the samsung snow chromebook (XE303C12) does not boot in the ARM Chromebook snow. Is the image damaged ? 

### ISSUE THREAD

**Issue by Marietto2008** [hexdump0815/linux-mainline-on-arm-chromebooks#9]
Hello.

I've dd'ed this ubuntu image to one sd card :

https://github.com/hexdump0815/imagebuilder/releases/download/191230-01/chromebook_snow-armv7l-ubuntu.img.gz

and I suspect that it is damaged,since this is what happens when I try to boot it in my ARM Chromebook snow :

![Untitled](https://github.com/hexdump0815/linux-mainline-on-arm-chromebooks/assets/755339/e24448d0-6b85-4e5c-bb7c-e38b10c9e064)

**Comment by rakslice** [comment]
this pxe stuff is boot from lan => not booting from sd at all?
is this the issue mentioned in the notes where newer/bigger sd cards don't work?

**Comment by hexdump0815** [comment]
yes the pxe output is misleading and just shows that u-boot cannot read the sd card and tries the next boot option which is pxe in this case which does not make sense on this device ... pxe boot support could be removed in future u-boot versions for it to avoid this confusion
